MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1145 @@
+---
+layout: post
+title: Data Editors in .NET MAUI DataForm(SfDataForm) control | Syncfusion
+description: Learn about the supported data editors in Syncfusion .NET MAUI DataForm (SfDataForm) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfDataForm
+documentation: UG
+---
+
+# Editors in .NET MAUI DataForm (SfDataForm)
+
+The data form supports the following built-in editors to edit the data:
+
+<table>
+<tr>
+<th>Editor name</th>
```

Comment:
Include a column

<th>Generated DataFormItem Type</th>

And update details for all editors like this

https://github.com/syncfusion-content/maui-docs/pull/688/files#diff-03dc9364f7e99b5b516a60453226f82544c74b292bb4fdd61ac27ed647f43913R27

Reply:
Column order

<th>Editor name</th>
<th>Supported Data Type/Attribute</th>
Generated DataFormItem Type
<th>Input control loaded</th>

Reply:
new column added